industrial_ci/_wrap_test.sh

```diff
@@ -19,9 +19,15 @@
 
 set -e # exit script on errors
 
+# run shellcheck for all files
+files=($(find . -name '*.sh'))
+echo "shellcheck ${files[*]}"
+shellcheck -sbash "${files[@]}"
```

Comment:
I might have missed discussion but you seem to have dropped running `shellcheck` in ICI in the subsequent PRs. Just curious what the decision was?

Reply:
I figured it's [run in .travis.yml](https://github.com/ros-industrial/industrial_ci/pull/369/files#diff-354f30a63fb0907d4ad57269548329e3R84).

Reply:
It got merged in #369, which you approved as well ;)